tests/driver_si70xx/Makefile

```diff
@@ -1,6 +1,8 @@
 include ../Makefile.tests_common
 
-# This test should also work with Si7006, Si7013 and Si7020 variants.
-USEMODULE += si7021
+# This test should also work with si7006, si7013, si7020 and si705x variants.
+SI70XX_VARIANT ?= si7021
```

Comment:
What's the use of this `SI70XX_VARIANT`? I only see it used here, so why not use `USEMODULE` directly?

Reply:
This is so you can test from the command line on a board even if you don't have the si7021, as mentioned in the Testing procedure section `make SI70XX_VARIANT=si705x -C tests/driver_si70xx/ all flash`. Probably for this specific Makefile you can set call `make USEMODULE=si705x ...` (but not `USEMODULE=si705x make ...`), but that's only because there are no other modules listed in the USEMODULE line.